crates/storage/src/vec.rs

```diff
@@ -13,6 +13,16 @@ pub struct VecStorage<T> {
     dims: u16,
 }
 
+impl<T> VecStorage<T> {
+    pub fn new(vectors: MmapArray<T>, payload: MmapArray<Payload>, dims: u16) -> Self {
+        Self {
+            vectors,
+            payload,
+            dims,
+        }
+    }
+}
+
```

Comment:
It should be gated by a cargo feature too.

Reply:
I'm not sure whether my use of cargo feature is standard and elegant, PTAL